docs/use/api.rst

```diff
@@ -155,23 +155,41 @@ following:
 -   Retries :ref:`temporary download errors
     <zyte-api-temporary-download-errors>` up to 3 times.
 
--   Retries network errors for up to 15 minutes.
+-   Retries network errors until they have happened for 15 minutes straight.
 
 All retries are done with an exponential backoff algorithm.
 
+.. _aggressive-retry-policy:
+
 If some :ref:`unsuccessful responses <zyte-api-unsuccessful-responses>` exceed
 maximum retries with the default retry policy, try using
-:data:`~zyte_api.aggressive_retrying` instead. Alternatively, the reference
-documentation of :class:`~zyte_api.RetryFactory` and
-:class:`~zyte_api.ConvervativeRetryFactory` features some examples of custom
+:data:`~zyte_api.aggressive_retrying` instead, which modifies the default retry
+policy as follows:
+
+-   Temporary download error are retried 7 times. :ref:`Permanent download
+    errors <zyte-api-permanent-download-errors>` also count towards this retry
+    limit.
+
+-   Retries permanent download errors up to 3 times.
+
+-   Retries error responses with an HTTP status code in the 500-599 range (503,
+    520 and 521 excluded) until they have happened 4 times in a row, not
+    counting rate-limiting responses or network errors that may happen in
+    between.
+
+Alternatively, the reference documentation of :class:`~zyte_api.RetryFactory`
+and :class:`~zyte_api.AggressiveRetryFactory` features some examples of custom
 retry policies, and you can always build your own
 :class:`~tenacity.AsyncRetrying` object from scratch.
 
-To use a custom retry policy, pass it when creating your client object:
+To use :data:`~zyte_api.aggressive_retrying` or a custom retry policy, pass it
+when creating your client object:
```

Comment:
I recently implemented a custom retry policy (basically, to disable all retries), following the docs. I made a mistake: configured scrapy-zyte-api setting with a path to a Retrying class (like for 99% other Scrapy settings). It lead to weird behavior - the spider started, and then all requests started to fail with an obscure error.

Do you think it's possible to be more explicit in the docs here, about classes vs objects? 

For example, here we write "custom retry policy". I'd expect that if I subclass an existing policy, and override it, it's my custom retry policy - but that's not the case, you also need to create an instance.

Reply:
I’m not sure how to make it more explicit here. Do you have some ideas?

Alternatively, maybe we could instead try to detect such cases at the client side and raise an exception early with a helpful message.

Reply:
Helpful message would definitely help. I also think we should raise an exception much earlier, and try to prevent spider from running if possible, if it happens.

Here, maybe we can write "pass an instance of your AsyncRetrying subclass" instead of "pass it"?